bindings/go/plugin/go.mod

```diff
@@ -10,8 +10,8 @@ require (
 	ocm.software/open-component-model/bindings/go/blob v0.0.9
 	ocm.software/open-component-model/bindings/go/configuration v0.0.9
 	ocm.software/open-component-model/bindings/go/constructor v0.0.0-20251024100521-a7be1bf5ce0d
-	ocm.software/open-component-model/bindings/go/credentials v0.0.2
-	ocm.software/open-component-model/bindings/go/descriptor/runtime v0.0.0-20251114112331-3af5ed79952f
+	ocm.software/open-component-model/bindings/go/credentials v0.0.3-0.20251114112331-3af5ed79952f
```

Comment:
Still waiting for https://github.com/open-component-model/open-component-model/actions/runs/19364724866